cylc/flow/remote.py

```diff
@@ -138,9 +138,30 @@ def run_cmd(
             return True
 
 
-def construct_ssh_cmd(raw_cmd, user=None, host=None, forward_x11=False,
-                      stdin=False, ssh_login_shell=None, ssh_cylc=None,
-                      set_UTC=False, allow_flag_opts=False):
+def construct_platform_ssh_cmd(raw_cmd, platform, **kwargs):
+    """A wrapper around `construct_ssh_cmd` allowing us to pass a platform
+    object rather than a user and host.
+
+    Args:
+        All as `construct_ssh_cmd` except for user and host.
+    """
+    ret = construct_ssh_cmd(
+        raw_cmd,
+        user=platform['owner'],
```

Comment:
I think we've killed the `owner` field now?

Reply:
Yay!

Reply:
Can remove this field from here then.